app/src/lib/open-shell.ts

```diff
@@ -20,6 +20,11 @@ export function openShell(fullPath: string, shell?: string) {
     return spawn('gnome-terminal', commandArgs, { shell: true })
   }
 
+  if (__LINUX__) {
```

Comment:
This looks identical to the `if` block above, so it probably shouldn't be here ;)